src/mono/wasm/debugger/DebuggerTestSuite/DebuggerTestBase.cs

```diff
@@ -130,19 +130,19 @@ static DebuggerTestBase()
                 Directory.Delete(TempPath, recursive: true);
         }
 
-        public DebuggerTestBase(ITestOutputHelper testOutput, string driver = "debugger-driver.html")
+        public DebuggerTestBase(ITestOutputHelper testOutput, string driver = "debugger-driver.html", string locale = "en-US")
```

Comment:
don't set the default value for `locale` here. We can depend on it being set by the derived classes.